data-model/fdo-type/term/term.json

```diff
@@ -0,0 +1,195 @@
+            "http://www.w3.org/2000/01/rdf-schema#Class",
+            "http://www.w3.org/2002/07/owl#Class",
+            "http://www.w3.org/2004/02/skos/core#Concept"
+          ]
+        }
+      }
+    },
+    "dcterms:identifier": {
+      "type": "string",
+      "description": "Handle of the Term",
+      "pattern": "^https://hdl\\.handle\\.net/[\\w.]+/(.){3}-(.){3}-(.){3}",
+      "examples": [
+        "https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"
+      ]
+    },
+    "ods:type": {
+      "type": "string",
+      "description": "The DOI to the FDO type of the object",
+      "pattern": "^https://doi\\.org/[\\w.]+/[\\w.]+",
+      "examples": [
+        "https://doi.org/10.15468/1a2b3c"
+      ]
+    },
+    "ods:status": {
+      "type": "string",
+      "enum": [
+        "ods:Draft",
+        "ods:Active",
+        "ods:Tombstone"
+      ],
+      "description": "The status of the Digital Object",
+      "examples": [
+        "ods:Active"
+      ]
+    },
+    "schema:version": {
+      "type": "integer",
+      "minimum": 1,
+      "description": "Version of the Schema, https://schema.org/version",
+      "examples": [
+        1
+      ]
+    },
+    "schema:name": {
+      "type": "string",
+      "description": "Name of the Term, https://schema.org/name",
+      "examples": [
+        "ods:DigitalSpecimen"
+      ]
+    },
+    "skos:prefLabel": {
+      "type": "string",
+      "description": "The preferred lexical label for the term",
+      "examples": [
+        "Digital Specimen"
+      ]
+    },
+    "schema:dateCreated": {
+      "type": "string",
+      "description": "Timestamp of creation. Internally generated, https://schema.org/dateCreated",
+      "format": "date-time",
+      "examples": [
+        "2021-06-01T12:00:000Z"
+      ]
+    },
+    "schema:dateModified": {
+      "type": "string",
+      "description": "Timestamp of last modification. Internally generated, https://schema.org/dateModified",
+      "format": "date-time",
+      "examples": [
+        "2021-06-06T13:22:00.445Z"
+      ]
+    },
+    "schema:creator": {
+      "type": "object",
+      "description": "Contains an ods:Agent object",
+      "$ref": "https://schemas.dissco.tech/schemas/fdo-type/shared-model/0.3.0/agent.json",
+      "examples": [
+        {
+          "@id": "https://orcid.org/0000-0002-1825-0097",
+          "@type": "schema:Person",
+          "schema:name": "John Doe",
+          "schema:email": "john.doe@example.com"
+        }
+      ]
+    },
+     "schema:contributor": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "List of contributors",
+      "examples": [
+        ["Jane Smith", "Bob Johnson"]
+      ]
+    },
+    "skos:definition": {
+      "type": "string",
+      "description": "A definition of what the term represents.",
+      "examples": [
+        "A digital representation of a physical specimen"
+      ]
+    },
+   
+    "dcterms:licence": {
```

Comment:
Mixing schema and dcterms. I would say stick with one or the other but mixing might be confusing. Most look to be coming from schema then I would suggest https://schema.org/license